Dockerfile

```diff
@@ -18,6 +18,8 @@ SVpFCiAgICAgICAgcmVxdWlyZWQ6IFRydWUKICAgICAgICBkZWZhdWx0OiAxMAogICAgICAgIHR5\
 cGU6IG51bWJlcgogICAgICAgIHRpdGxlOiBQcm9tZXRoZXVzIFN0b3JhZ2UgU2l6ZSAoR2IpCg=="
 
 
+
```

Comment:
Like this change. We should do more changes like this.

Reply:
This is a bug in the apb tool. It will insert a newline for every build.

Reply:
Ha, didn't catch this.

Reply:
@pb82 You made my day! With your note my garbage comment actually make sense now